docs/build.md

```diff
@@ -1,48 +1,52 @@
 # Building onos-config
+In order the build the project, developers are expected to install the 
+required [development tools](prerequisites.md). 
 
-> Steps outlined in this page assume that the [development prerequisites](prerequisites.md) are met. 
-
-## Building Locally
-
-The project provides a `Makefile` that supports all the  required to unit test,
-validate and build the project.
-The makefile supports the following targets:
-* `test` - pulls dependencies and runs tests, including lint, vet and license
-* `build` - runs `protos`, `test` and builds an executable
-* `all` - runs `protos`, and creates the run-time `onosproject/onos-config` Docker image
-  * Requires a local Docker daemon
-
-## Building via Docker Developer Image
-
-Since the build requires a number of tools for depdency generation, linting, vetting, and 
-compiling, the project provides a published Docker image `onosproject/golang-build`, which 
-includes all the required facilities to allow developers to build the project with minimum 
-requirements placed on their own machines.
- 
-To build with the build image, mount the onos-config root to the container
-and pass the desired argument to the Makefile entrypoint of the developer image as follows
-(from the onos-config directory of your development machine):
+Currently, the project build and validation is driven by a top-level `Makefile`, which supports the following usage:
 ```bash
-docker run -it -v `pwd`:/go/src/github.com/onosproject/onos-config onosproject/golang-build:stable build
+> make help
+build                           build the Go binaries and run all validations (default)
+clean                           remove all the build artifacts
+coverage                        generate unit test coverage data
+deps                            ensure that the required dependencies are in place
+gofmt                           run the Go format validation
+images                          build all Docker images
+license_check                   examine and ensure license headers exist
+lint                            run the linters for Go source code
+onos-cli-docker                 build onos-cli Docker image
+onos-config-docker              build onos-config Docker image
+protos                          compile the protobuf files (using protoc-go Docker)
+run-docker                      run onos-config docker image
+test                            run the unit tests and source code validation
+vet                             examines Go source code and reports suspicious constructs
 ```
 
-See [run.md](run.md) for details on how to run the applications.
```

Comment:
might still be needed ?

Reply:
I don't think so. The build page is now fairly concise and it should not be difficult to back out and go to the 'how to run' page.